### PR TITLE
Job Pass Rates By Most Reduced Pass Rate

### PR DESCRIPTION
**Pull request by deads2k**
Adds slicing for jobs that suddenly start failing based on the last two days of results.  This is currently horrifying....

Screenshot from 2020-09-15 16-54-21

### CODE REVIEW

File: pkg/html/fastest_moving_jobs.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	`)
+
+	jobDisplayed := 0
+	for _, jobDetails := range jobPassChanges {
+		jobDisplayed++
+		if jobDisplayed > 10 {
+			break
+		}
+		// don't display things moving in the right direction
+		if jobDetails.passPercentageChange > 0 {
```
Comment (bparees) [inline pkg/html/fastest_moving_jobs.go:50]:
```suggestion
		if jobDetails.passPercentageChange > 10 {
```

File: pkg/html/fastest_moving_jobs.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		</tr>
+	`)
+
+	jobDisplayed := 0
+	for _, jobDetails := range jobPassChanges {
+		jobDisplayed++
+		if jobDisplayed > 10 {
+			break
+		}
+		// don't display things moving in the right direction
```
Comment (bparees) [inline pkg/html/fastest_moving_jobs.go:49]:
```suggestion
		// don't display things moving in the right direction or that only dropped within the margin of error
```

File: pkg/html/fastest_moving_jobs.go (hunk 1 above)
Comment (deads2k) [reply pkg/html/fastest_moving_jobs.go]: updated

**Pull request merged**